pkgs/development/python-modules/autopep8/default.nix

```diff
@@ -1,20 +1,26 @@
 { lib
 , buildPythonPackage
 , fetchFromGitHub
+, fetchpatch
 , glibcLocales
 , pycodestyle
 , pytestCheckHook
 , pythonOlder
+, setuptools
 , tomli
 }:
 
 buildPythonPackage rec {
   pname = "autopep8";
   version = "2.0.4";
-  format = "setuptools";
+  pyproject = true;
 
   disabled = pythonOlder "3.7";
 
+  build-system = [
+    setuptools
+  ];
+
```

Comment:
Please move this below src

Reply:
Done. Thanks a lot, I will be more careful regarding the order in the future